lib/src/repositories/checking_device_online_repository.dart

```diff
@@ -41,20 +44,48 @@ class CheckingDeviceOnlineRepository {
     });
   }
 
+  /// We want to read these sequentially for now, some errors observed trying to read ble characteristics in parallel
```

Comment:
probably do want to add a lock to the provisioning package for reading/writing ble characteristics (follow-up..)